plugins/LadspaEffect/swh/CMakeLists.txt

```diff
@@ -61,8 +62,8 @@ FOREACH(_item ${XML_SOURCES})
 	endif(LMMS_BUILD_WIN32)
 
 	SET_TARGET_PROPERTIES("${_plugin}" PROPERTIES PREFIX "")
```

Comment:
@DomClark is it fine to remove this line? from what i understand, this line seems to be a no-op

Reply:
Nvm looked at it now. Seems to be needed

Reply:
Right ,windows doesn't use the `lib` prefix for libraries.  For example, `sndfile` is `libsndfie` on Linux, but `sndfile.dll` on Windows.